view/frontend/web/js/view/payment/method-renderer/creditcard.js

```diff
@@ -107,6 +108,10 @@ define(
                     return this;
                 },
 
+                getCreditcardType: ko.observable(function () {
+                    return this.creditcardIssuer;
+                }),
```

Comment:
what's the use of this observable?

Reply:
it is used in <div data-bind="attr: { 'class': 'buckaroo_magento2_all_creditcard payment-method buckaroo_magento2_' + code }, css: {'_active': ($parent.getCode() == $parent.isChecked() && **$parent.getCreditcardType()** == code)}">

This adds a CSS class _active if the credit card type returned by getCreditcardType() matches the code variable

Reply:
why not use directly `selectedCard()` method?